quactography/solver/io.py

```diff
@@ -37,7 +37,7 @@ def save_optimization_results(
     None
     """
     np.savez(
-        outfile,
+        "data/output_graphs/"+outfile,
```

Comment:
There should not be a hardcoded path like this in a function. It should be given in `outfile`. For instance, if your previous `outfile` was "test", it should become "data/output_graphs/test". Deal with it in the script, and even in the script try to avoid hardcoded paths. The input of the script should be able to handle a path for `outfile`.